components/list/test/list.vdiff.js

```diff
@@ -285,11 +286,11 @@ describe('list', () => {
 			</d2l-list>
 		`;
 		function createSelectableList(opts) {
-			const { selected, addButton } = { selected: false, addButton: false, ...opts };
+			const { selected, addButton, selectionDisabled } = { selected: false, addButton: false, selectionDisabled: false, ...opts };
 			return html`
 				<d2l-list style="width: 400px;" ?add-button="${addButton}">
 					<d2l-list-item label="Item 1" selectable key="1" ?selected="${selected}" color="${ifDefined(!selected ? '#00ff00' : undefined)}">Item 1</d2l-list-item>
-					<d2l-list-item label="Item 2" selection-disabled selectable key="2" ?selected="${selected}" color="${ifDefined(selected ? '#00ff00' : undefined)}">Item 2</d2l-list-item>
+					<d2l-list-item label="Item 2" ?selection-disabled="${selectionDisabled}" selectable key="2" color="${ifDefined(selected ? '#00ff00' : undefined)}">Item 2</d2l-list-item>
```

Comment:
wanted to be able to dynamically set this so that hover/focus of sibling was possible for new tests